src/Pages/ExplorePage.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "./../components/Navbar";
+import ExplorePageBody from "./../components/Explore Page/ExplorePageBody";
+
+const ExplorePage = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
If `Navbar` is common in every component then you can import it in `App.js` to reduce redundancy in the code